src/ray/protobuf/serve.proto

```diff
@@ -246,8 +246,9 @@ enum DeploymentStatus {
   DEPLOYMENT_STATUS_UPDATING = 0;
   DEPLOYMENT_STATUS_HEALTHY = 1;
   DEPLOYMENT_STATUS_UNHEALTHY = 2;
-  DEPLOYMENT_STATUS_UPSCALING = 3;
```

Comment:
Non-blocker: I think generally it's not good to reorder fields in protobufs, but probably fine in this case.

Reply:
yeah agree, although in this case I think it's fine.